tests/benchmarks/bench_sched_flags/src/main.rs

```diff
@@ -0,0 +1,28 @@
+#![no_main]
+#![no_std]
+#![feature(type_alias_impl_trait)]
+#![feature(used_with_arg)]
+
+use riot_rs::{
+    debug::log::*,
+    thread::{thread_flags, ThreadId},
+};
+
+#[riot_rs::thread(autostart)]
+fn thread0() {
+    match riot_rs::bench::benchmark(1000, || {
+        thread_flags::set(ThreadId::new(1), 1);
```

Comment:
the `ThreadId`s are not necessarily 0 and 1. This will e.g., show bogus results when the target board starts some other thread for whatever reason.
Can `embassy_sync::OnceLock` be used with `block_on` in threads?

Reply:
Done with a channel in d8a5bed.